src/linux/cpu.rs

```diff
@@ -531,6 +531,202 @@ pub(crate) fn get_physical_core_count() -> Option<usize> {
     Some(core_ids_and_physical_ids.len())
 }
 
+fn get_arm_implementer(implementer: u16) -> Option<&'static str> {
```

Comment:
Can you add a comment in the code saying where you got the value (the link you posted in the PR comment 😉 ) please?